cli.py

```diff
@@ -167,6 +190,10 @@ class Meta:
         arguments = [
             (['name'],
              dict(help='The item name.', action='store')),
+            (['-a', '--account'],
```

Comment:
I set a `COMMON_ARGS` to include `--account` and `--vault` as global arguments. Cement doesn't provide a better approach. I took a look and It has a newest alternative to the `CementBaseController` that could help, but it changes everything, and I don't like it so far.